src/ray/core_worker/test/direct_actor_transport_test.cc

```diff
@@ -821,11 +821,6 @@ class MockTaskReceiver : public TaskReceiver {
                      task_handler,
                      initialize_thread_callback,
                      actor_creation_task_done_) {}
-
-  void UpdateConcurrencyGroupsCache(const ActorID &actor_id,
-                                    const std::vector<ConcurrencyGroup> &cgs) {
-    concurrency_groups_cache_[actor_id] = cgs;
-  }
```

Comment:
I think this might be used by Java, which runs multiple actors in the same process. Can you check?

Reply:
oh sorry I was just commenting on the PR in general, not this util

Reply:
> which runs multiple actors in the same process

Interesting, I am wondering whether this is possible or not because `DestroyActor` is called when an actor is out of scope. When GCS calls `DestroyActor`, the core process will release local references, kill child processes, disconnect with raylet, ... etc.

Reply:
My knowledge here is really out of date, so you might be right, but from what I remember:

There is a distinction between the `CoreWorker` and the `CoreWorkerProcess`. These are 1-1 in python but N-1 in java. When the GCS calls `DestroyActor`, I believe i destroys the `CoreWorker`, but not the `CoreWorkerProcess` in java.

Reply:
Wow, interesting. Thank you for providing the context!